### ISSUE TITLE
Fix reported name for Python modules in subdirs

### ISSUE THREAD

**Issue by adriaandegroot** [calamares/calamares#1641]
This applies to the CMake parts that are used by *calamares-extensions* repo (which are installed from the Calamares repo, which is why the issue is here). The idea is that for external modules (e.g. those in *calamares-extensions*) there are simple / usable CMake macros to build a module.

In the extensions repo, the report on modules looks like this:

```
-- Found  module: mobile
   TYPE: viewmodule
   LINK_LIBRARIES: 
   LINK_PRIVATE_LIBRARIES: 
   PLUGIN_DESTINATION: lib/calamares/modules/mobile
   CONFIGURATION_FILES: mobile.conf => [Build directory only]
   RESOURCES: mobile.qrc

-- Found  module: modules/slowpython
   TYPE: jobmodule
   MODULE_DESTINATION: lib/calamares/modules/slowpython
   CONFIGURATION_FILES: slowpython.conf => [Build directory only]
```

Notice that the name, e.g. the bit after `Found module:`, is a bit odd: *modules/slowpython* is not actually the name, there's an extra *modules/* in there. This name confusion does not show up with other modules in the extensions repo, so it is probably related to being-a-Python-module.

So the issue / feature is: improve the reported "name" of a (Python) module when it lives in a subdirectory, to be only the last part of the directory path / the actual module name.

**Comment by adriaandegroot** [comment]
To test, you could build and install Calamares (in a VM or whatever):

```
cd calamares
cmake . -DCMAKE_INSTALL_PREFIX=/tmp/calamares -DINSTALL_POLKIT=OFF
make -j4 install
```

Then configure the calamares-extensions repo:

```
cd calamares-extensions
cmake . -DCMAKE_INSTALL_PREFIX=/tmp/calamares
```

This will show the "wrong" module name. You can then edit the **installed** CMake helper files to fix the problem, just re-running cmake for calamares-extensions; once fixed you can copy the modified CMake helper files back to the calamares repo and file a PR.

**Comment by adriaandegroot** [comment]
Fixed in 0bf1df72cdc023a5d1ca1c3ec945ce6d71ec2626
